<launch>
    <!-- <arg name="markerSize"      default="0.025"/>
    <arg name="markerWidth"     default="8"/>
    <arg name="markerHeight"     default="11"/>
    <arg name="marker_frame"    default="chessboard_marker_frame"/>
    <arg name="ref_frame"       default="camera_color_optical_frame"/>
    <node pkg="chessboard" type="single" name="single" output="screen">
        <remap from="/camera_info" to="/camera/color/camera_info" />
        <remap from="/image" to="/camera/color/image_raw" />
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="marker_width"        value="$(arg markerWidth)"/>
        <param name="marker_height"        value="$(arg markerHeight)"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>
        <param name="camera_frame"       value="camera_color_optical_frame"/>
        <param name="marker_frame"       value="$(arg marker_frame)" />
    </node> -->
            <arg name="markerSize"      default="0.015"/>
        <arg name="markerWidth"     default="8"/>
        <arg name="markerHeight"     default="11"/>
        <arg name="marker_frame"    default="chessboard_marker_frame"/>
        <arg name="ref_frame"       default="mechmind"/>

    <node pkg="chessboard" type="single" name="single" output="screen">
        <remap from="/camera_info" to="/mechmind/color/camera_info" />
        <remap from="/image" to="/mechmind/color/image_raw" />
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="marker_width"        value="$(arg markerWidth)"/>
        <param name="marker_height"        value="$(arg markerHeight)"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>
        <param name="camera_frame"       value="mechmind"/>
        <param name="marker_frame"       value="$(arg marker_frame)" />
    </node>
</launch>